### DISCUSSION TITLE
be a member

### DISCUSSION THREAD

**Discussion by Jaures-beinjamin** [shnug/jobs#42] [General] [answered]
how I can be a member and contribute I am co-founder of the dotnet Cameroon community from Cameroon in Africa

**Comment by tonyqus** [comment, marked as answer]
Hi, we only have wechat group for now (no Discord server). Do you have wechat installed on your phone?

**Reply by Jaures-beinjamin** [reply]
Yes +237675257302

**Reply by tonyqus** [reply]
Is your name on Wechat 'Jaures'?

**Reply by tonyqus** [reply]
And we mainly speak Simplified Chinese in the wechat group. Is this fine with you?

**Reply by Jaures-beinjamin** [reply]
UHMM I DON’T UNDERSTAND CHINESE English write yes

**Reply by JohnNt97** [reply]
我也想加一下，要在这里写上微信号吗

**Reply by tonyqus** [reply]
https://github.com/dotnet-cn/jobs/issues/13

**Comment by webclient163** [comment]
用中文当然更方便啊，我的英语很生硬...
















在 2024-01-31 17:00:23，"Tony Qu" ***@***.***> 写道：

And we mainly speak Simplified Chinese in the wechat group. Is this fine with you?
